### ISSUE TITLE
UDP service scan always triggers warning "Your ports include "T:" but you haven't specified any TCP scan type".

### ISSUE THREAD

**Issue by cnotin** [nmap/nmap#2135]
**Describe the bug**
I run a simple UDP port and service scan and I see this warning:
> Your ports include "T:" but you haven't specified any TCP scan type
Even though I did not specify any port ranges, even less with a "T:"

**To Reproduce**
```
nmap -sUV -v 192.168.0.1
[...]
Completed UDP Scan at [...]
Your ports include "T:" but you haven't specified any TCP scan type
Initiation Service scan at [...]
```

**Expected behavior**
This warning should not appear.

**Version info (please complete the following information):**
 - OS: Windows or Kali
 - Output of `nmap --version`:
```
Nmap version 7.90 ( https://nmap.org )
Platform: i686-pc-windows-windows
Compiled with: nmap-liblua-5.3.5 openssl-1.1.1h nmap-libssh2-1.9.0 nmap-libz-1.2.11 nmap-libpcre-7.6 Npcap-1.00 nmap-libdnet-1.12 ipv6
Compiled without:
Available nsock engines: iocp poll select
```

**Additional context**
The message is from `getpts_aux()` so I have added a `printf` for `origexpr` and it appears that the offending port filter is `"T:9100-9107` which we can found in the `nmap-service-probes` files to exclude those TCP ports! I suppose that even though I did not specificy any port range, those were added silently by Nmap to exclude them (to not scan printer ports I guess).

**Comment by cnotin** [comment]
Thanks @bonsaiviking  :)

**Comment by fyodor** [comment]
Update: this is now fixed in [Nmap 7.91](https://seclists.org/nmap-announce/2020/2)

**Comment by goarblt** [comment]
hello
https://www.evil.com/
